### PR TITLE
🌱 Add release notes pr statistics

### PR DESCRIPTION
**Pull request by oscr**
**What this PR does / why we need it**:

Automates the fun stats section of the weekly and release notes. 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:

cc @joekr @sayantani11 @VibhorChinda

### CODE REVIEW

**Comment by sbueringer** [comment]
nice!

/lgtm

File: hack/tools/release/notes.go [hunk 1]
```diff
@@ -204,6 +204,12 @@ func run() int {
 	// TODO Turn this into a link (requires knowing the project name + organization)
 	fmt.Printf("Changes since %v\n---\n", commitRange)
 
+	fmt.Printf("## Fun stats\n")
+	fmt.Printf("%d PRs merged into release\n", len(commits))
```
Comment (killianmuldoon) [inline hack/tools/release/notes.go:208]: Maybe remove "release here" to make this line up with the other outputs?

**Pull request merged**